src/Microsoft.DocAsCode.Plugins/DocumentException.cs

```diff
@@ -10,26 +10,17 @@ public class DocumentException : Exception
         public string File { get; set; }
         public int Line { get; set; }
         public int Column { get; set; }
-        public string Code { get; set; }
 
         public DocumentException() { }
         public DocumentException(string message) : base(message) { }
-        public DocumentException(string message, string code) : this(message)
```

Comment:
the removal of these methods would bring breaking changes to plugins - shall we just keep these interfaces?

Reply:
I have updated the plugins to use DocumentException(string message).